Horde3D/Source/ColladaConverter/converter.cpp

```diff
@@ -1232,6 +1232,9 @@ bool Converter::writeSceneGraph( const string &assetPath, const string &assetNam
 		outf << "\t<!-- Morph targets: ";
 		for( unsigned int i = 0; i < _morphTargets.size(); ++i )
 		{
+			char msg[1024];
+			sprintf( msg, "Added Morph target: '%s'",  _morphTargets[i].name);
+			log( msg, true );
```

Comment:
log function receives std::string, so you may use something like that:
`log( "Found morph controller with -morph for requested id " + id + "\n", true );`
sprintf will likely trigger a warning on MSVC.

Reply:
I did try that at first but mingw didn't like it with .name for some reason, I saw sprintf being used for logging, so I copied it.
I will try it agian tho

Reply:
Yep, I've tried with gcc, it gives error. You can try like this:
            log( std::string( "Added morph target: " ) + _morphTargets[i].name );